lib/taskana-core/src/main/java/pro/taskana/TaskanaEngine.java

```diff
@@ -98,6 +98,11 @@
      */
     void checkRoleMembership(TaskanaRole... roles) throws NotAuthorizedException;
 
+    /**
+     * Check if the database used is postgreSQL and do the rollback.
+     */
+    void rollbackToPostgreSQL();
```

Comment:
This method is no longer required, is it?

Reply:
this method has been removed